### DISCUSSION TITLE
How to disable the nag about "logs have grown to 3 files" without deleting the logs?

### DISCUSSION THREAD

**Discussion by CalculonPrime** [Rem0o/FanControl.Releases#2032] [Q&A]
I'm getting nagged about "logs have grown to 3 files" but I have plenty of disk space so I just want to suppress the nag screen.

**Comment by Rem0o** [comment]
Why would you not delete the logs? Otherwise, just create an "archive" folder and move them there.

**Comment by CalculonPrime** [comment]
Well, with Windows itself, it just keeps event logs until you delete them.  I guess the thinking is that you might not know you need to look back at events until some problem happens and at that point, it's better to have the events and not need them then need them and not have them.

I measured the rate of your logging and its about 100KB per day. At that rate it would take 27 years to use of 1GB (and I have 1.7TB free on my system drive).  So it would take 45900 years for FanControl's logging to use up all available space on my drive.

Maybe you could create some config option called "maxLogFiles", and then set it 3 but users could change it.  So if I want 27 years worth of logging, I won't get nagged for it! :-)
